packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -186,6 +198,51 @@ export class InteractionTracker {
     });
   }
 
+  private interceptXhrTasks(task: AsyncTask) {
+    if (isTrackedTask(task) && task.target instanceof XMLHttpRequest) {
```

Comment:
Optional: you can reduce the nesting of this function by negating the `if` and making it return early, e.g.:

```
if (!isTrackedTask(task)) return;
if (!task.target instanceof XMLHttpRequest) return;

const xhr = ...
```